extension/httpfs/httpfs.cpp

```diff
@@ -36,6 +36,8 @@ HTTPParams HTTPParams::ReadFrom(FileOpener *opener) {
 	float retry_backoff = DEFAULT_RETRY_BACKOFF;
 	bool force_download = DEFAULT_FORCE_DOWNLOAD;
 	bool keep_alive = DEFAULT_KEEP_ALIVE;
+	bool enable_server_cert_verification = DEFAULT_ENABLE_SERVER_CERT_VERIFICATION;
+	std::string_view ca_cert_file = DEFAULT_CA_CERT_FILE;
```

Comment:
I dont think this should be a string_view, string is fine

Reply:
Thanks for the feedback. Fixed.